### PR TITLE
Use tf2xla helpers to implement convolution gradients

### PR DESCRIPTION
**Pull request by asuhan**
The precision config change landed in TF, so we can now use them again.

### CODE REVIEW

File: torch_xla/csrc/convolution.cpp [hunk 1]
```diff
@@ -9,232 +11,80 @@
...
-  xla::Shape weight_shape = XlaHelpers::ShapeOfXlaOp(weight);
-  return xla::Pad(
-      xla::ConvGeneralDilated(grad_output, mirrored_weights,
-                              /*window_strides=*/ones, padding, lhs_dilation,
-                              rhs_dilation, dnums,
-                              /*feature_group_count=*/1,
-                              /*batch_group_count=*/1, &precision_config),
-      XlaHelpers::ScalarValue<float>(0, weight_shape.element_type(), builder),
-      padding_config);
+  return tensorflow::MakeXlaBackpropInputConvOp(
```
Comment (dlibenzi) [inline torch_xla/csrc/convolution.cpp]: Use our Consume(), which throws instead of dying.

File: torch_xla/csrc/convolution.cpp [hunk 2]
```diff
@@ -9,232 +11,80 @@
...
+    tensorflow::gtl::ArraySlice<const xla::int64> spatial_padding) {
+  tensorflow::ConvOpAttrs conv_op_attrs =
+      MakeConvOpAttrs(spatial_stride, spatial_padding, {1, 1});
+  auto inv_transpose_permutation =
+      xla::InversePermutation(FilterTransposePermutation());
+  xla::Shape transposed_weight_shape = xla::ShapeUtil::PermuteDimensions(
+      inv_transpose_permutation, kernel_shape);
   xla::PrecisionConfig precision_config =
       XlaHelpers::BuildPrecisionConfig(XlaHelpers::mat_mul_precision());
+  xla::XlaOp conv = MakeXlaBackpropFilterConvOp(
```
Comment (dlibenzi) [inline torch_xla/csrc/convolution.cpp]: Consume()

File: torch_xla/csrc/convolution.cpp (hunk 1 above)
Comment (asuhan) [reply torch_xla/csrc/convolution.cpp]: Done.

File: torch_xla/csrc/convolution.cpp (hunk 2 above)
Comment (asuhan) [reply torch_xla/csrc/convolution.cpp]: Done.

**Pull request merged**